client/src/components/Draftail/Draftail.scss

```diff
@@ -1,4 +1,5 @@
 $draftail-editor-text: $color-input-text;
+$draftail-placeholder-text: theme('colors.grey.400');
```

Comment:
Unrelated bug fix – noticed while working on this we were using Draftail’s default placeholder text color rather than what we use elsewhere.